change/@microsoft-teams-js-98d7ca14-14e6-4abe-b79a-c08d1da0dc2c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Files upload API : 1. Changed datatype of `lastModified` field from Date to number 2. Added a new field `size` of type number",
```

Comment:
Please don't format this as a numbered list. This sentence shows up in public changelogs and the numbered list doesn't get formatted nicely. Try something like:

On the `File` interface changed the type of lastModified from `Date` to `number` and added a new field `size` that contains the size in bytes of the file.

Reply:
Will follow this now onwards. Updated the file.